src/ML/include/BipedalLocomotion/ML/velMANNAutoregressiveInputBuilder.h

```diff
@@ -0,0 +1,114 @@
+{
+public:
+    /**
+     * Constructor
+     */
+    velMANNAutoregressiveInputBuilder();
+
+    /**
+     * Destructor
+     */
+    ~velMANNAutoregressiveInputBuilder();
+
+    // clang-format off
+    /**
+     * Initialize the trajectory builder..
+     * @param paramHandler pointer to the parameters handler.
+     * @note the following parameters are required by the class
+     * |                  Parameter Name                 |   Type   |                                                             Description                                                             | Mandatory |
+     * |:-----------------------------------------------:|:--------:|:-----------------------------------------------------------------------------------------------------------------------------------:|:---------:|
+     * |                 `base_vel_norm`                 | `double` |                                              Norm of the desired base velocity in m/s                                               |    Yes    |
+     * |               `base_ang_vel_norm`               | `double` |                                              Norm of the desired base angular velocity in rad/s                                     |    Yes    |
+     * |             `ellipsoid_forward_axis`            | `double` |                                              Size of the forward axis ellipsoid in meters                                           |    Yes    |
+     * |            `ellipsoid_backward_axis`            | `double` |                                              Size of the backward axis ellipsoid in meters                                          |    Yes    |
+     * |              `ellipsoid_side_axis`              | `double` |                                              Size of the lateral axis ellipsoid in meters                                           |    Yes    |
+     * |            `ellipsoid_scaling_factor`           | `double` |                   Scaling factor considered in the generation of the final and control points of the Bezier Curve                   |    Yes    |
+     * |       `max_base_direction_angle_forward`      | `double` |                               Maximum angle of the base direction when the robot is walking forward                               |    Yes    |
+     * |      `max_base_direction_angle_backward`      | `double` |                               Maximum angle of the base direction when the robot is walking backward                              |    Yes    |
+     * | `max_base_direction_angle_side_opposite_sign` | `double` |         Maximum angle of the base direction when the robot is walking lateral and is facing towards the walking direction         |    Yes    |
+     * |   `max_base_direction_angle_side_same_sign`   | `double` | Maximum angle of the base direction when the robot is walking lateral and is facing towards the opposite of the walking direction |    Yes    |
```

Comment:
That's really a minor comment :D:. Can you align the columns XD

Reply:
Done in 75b78511.